src/app/(proper_react)/redesign/(authenticated)/user/dashboard/Dashboard.stories.tsx

```diff
@@ -98,12 +109,25 @@ const DashboardWrapper = (props: DashboardWrapperProps) => {
     createRandomScanResult({ status: "new", manually_resolved: true }),
   ];
 
+  const mockedOnlyUnresolvedScanResults: OnerepScanResultRow[] = [
```

Comment:
Heh, just for my interest: why does it matter that there are _only_ unresolved scan results, i.e. no resolved scan results? IIRC for the other tests it doesn't really matter, so another option would be to just remove the `...mockedResolvedScanResults` from `mockedUnresolvedScanResults`.

Reply:
This does not really matter to the broker scan-in-progress stories either. My thinking was that if there are exposures coming in during an initial scan only having unresolved ones is the more likely case. But in that case: Let’s keep it simple — removed in [9860df8](https://github.com/mozilla/blurts-server/pull/3431/commits/9860df837463406ce6ebcb16ec4b0323bbb2bb2f).